### PR TITLE
Add FBI Paranormal Investigator profession to Xedra Evolved

### PR DESCRIPTION
**Pull request by LilimDiam**
#### Summary
Content "Added x-files inspired profession to Xedra Evolved"

#### Purpose of change
Providing a starting profession that's both related to the mod and is not connected to XEDRA directly.

#### Describe the solution
A new profession that is a mix of police detective and XEDRA investigator.

#### Describe alternatives you've considered
N/A

#### Testing
Started a new game, spawned in as this profession, everything works as intended.

### CODE REVIEW

**Comment by bombasticSlacks** [comment]
image

File: data/mods/Xedra_Evolved/player/professions.json [hunk 1]
```diff
@@ -29,6 +29,46 @@
       { "item": "5x50_50_mag", "ammo-item": "5x50dart", "charges": 50 }
     ]
   },
+  {
+    "type": "profession",
+    "id": "fbi_investigator",
+    "name": "FBI Paranormal Investigator",
+    "description": "Before the Cataclysm, you used to work for Paranormal Investigation Department of FBI.  You got used to jokes at your expense, as well as being constantly underfunded and disregarded by your superiors.  You were returning from yet another case with nothing but a bag of suspicious candies to show, when all hell broke loose.  Oh well, at least now you can be certain that everyone still alive will belive you that the truth is out there.",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/player/professions.json:36]:
```suggestion
    "description": "Before the Cataclysm, you used to work for the Paranormal Investigation Department of the FBI.  You got used to jokes at your expense, as well as being constantly underfunded and disregarded by your superiors.  You were returning from yet another case with nothing but a bag of suspicious candies to show, when all hell broke loose.  Oh well, at least now you can be certain that everyone still alive will believe you that the truth is out there.",
```
(FBI Paranormal Investigation Department works too)

Comment (LilimDiam) [reply data/mods/Xedra_Evolved/player/professions.json:36]: Oh derp, articles are hard
Thanks for catching the typo too

**Review by Maleclypse — APPROVED** [review]
Thanks I love it!   I need to take a minute and add a bigfoot hunter profession.  She hunts with her camera.

**Pull request merged**